### ISSUE TITLE
SteamVR Beta Makes InputEmulator Unusable

### ISSUE THREAD

**Issue by blinkVR** [matzman666/OpenVR-InputEmulator#33]
It seems like with the update from Nov 27 the "under the hood" changes have broken the InputEmulator.

http://store.steampowered.com/news/externalpost/steam_community_announcements/2149769425930089711

The following doesn't seem to work when using Rift + Touch:

- Identifying controllers does not cause any haptic feedback
- You cannot create any offsets anymore, you cannot change any values in this submenu
- Clicking on "input remapping" causes InputEmulator to crash

**Comment by kblood** [comment]
Was about to report this one. Especially with Doom VFR where Rift users needs the newest beta, its a problem.

**Comment by ParadiseDecay** [comment]
Yes, just tried it with Doom VFR on Oculus + Touch and it does crash.

**Comment by kblood** [comment]
Well, for others like me that want a fix for Doom VFR regarding smooth locomotion I can recommend this:
https://www.reddit.com/r/oculus/comments/7h6nva/smooth_locomotion_in_doomvfr_for_rift_i_scripted/

**Comment by ParadiseDecay** [comment]
I'll give that a try thank you!

**Comment by romkslrqusz** [comment]
I am now having this even with the latest version of SteamVR, non-beta

**Comment by blinkVR** [comment]
@romkslrqusz I've just checked it with the latest SteamVR, non-beta update and it works fine for me.

**Comment by romkslrqusz** [comment]
@blinkVR only other variable I can think of would be an update to Oculus dash - I had Input Emulator stop working for me overnight, so something has to have changed
